src/caret_analyze/record/records_service/response_time.py

```diff
@@ -450,33 +450,46 @@ def __init__(
         self._histogram = ResponseHistogram(self._records, self._timeseries)
 
     def to_all_records(self) -> RecordsInterface:
-        return self._response_map_all.to_all_records()
+        """
+        Calculate the all records data for response time.
 
-    def to_worst_in_input_records(self) -> RecordsInterface:
-        return self._response_map_all.to_worst_in_input_records()
+        The all for response time are included message flow latency.
+
+        Returns
+        -------
+        RecordsInterface
+            Records of the all response time.
+
+            Columns
+            - {columns[0]}
+            - {'response_time'}
 
-    def to_stacked_bar(self) -> RecordsInterface:
         """
-        Calculate records for stacked bar.
+        return self._response_map_all.to_all_records()
+
+    def to_worst_case_records(self) -> RecordsInterface:
+        """
+        Calculate the worst case records data for response time.
```

Comment:
```suggestion
        Calculate data of the worst case records for response time.
```

Reply:
Fixed it in [a103900](https://github.com/tier4/CARET_analyze/pull/355/commits/a1039004f34c90c7c7f17675a466049aaf7220d5).